src/login/components/GroupManagement/Groups/CreateGroup.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component, Fragment } from "react";
+import InjectIntl from "../../../translation/InjectIntl_HOC_factory";
+import NameForm from "../NameForm";
+
+const CloseButton = () => (
+  <svg
+    className="close"
+    width="16"
+    height="16"
+    viewBox="0 0 16 16"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path d="M7 0h2v16H7z" />
+    <path d="M0 9V7h16v2z" />
+  </svg>
+);
+
+const WizardHeading = () => {
```

Comment:
The `WizardHeading` and `CreateGroupHeading` looks like a repetition of the same code with different text and button functions. Would it be an idea to combine this codes?

Reply:
:+1: